xpra-igv/Dockerfile

```diff
@@ -0,0 +1,119 @@
+# Add a default Connect client version. Can be overridden by build arg
+ARG CONNECT_CLIENT_VERSION="0.8.1-snapshot"
+
+# Path to the public connect-client
+FROM public.cr.seqera.io/platform/connect-client:${CONNECT_CLIENT_VERSION} AS connect
+
+FROM mambaorg/micromamba:1.5.10-noble AS micromamba
+
+FROM ubuntu:24.04
+
+ENV DEBIAN_FRONTEND=noninteractive
+
+ARG MAMBA_USER=root
+ARG MAMBA_USER_ID=0
+ARG MAMBA_USER_GID=0
+ENV MAMBA_USER=$MAMBA_USER
+ENV MAMBA_ROOT_PREFIX="/opt/conda"
+ENV MAMBA_EXE="/bin/micromamba"
+
+USER $MAMBA_USER
+
+RUN apt-get update --yes && apt-get install --yes --no-install-recommends \
+    apt-transport-https \
+    pciutils \
+    cups-filters \
+    cups-common \
+    cups-pdf \
+    ca-certificates \
+    git \
+    wget \
+    sudo \
+    libatomic1 \
+    python3.12 \
+    python3-cups \
+    python3-xdg \
+    python3-pyinotify \
+    python3-paramiko \
+    python3-netifaces \
+    python-is-python3 \
+    gnome-menus \
+    curl \
+    locales \
+    openjdk-17-jre \
+    gnupg \
+    kmod \
+    x11-xkb-utils \
+    dbus-x11 \
+    xvfb \
+    terminator && \
```

Comment:
```suggestion
    terminator && \
    thunar && \
```

Reply:
@robsyme Not sure we want both `thunar` and `terminator` lines to have double ampersand (`&&`) at the end, right? Isn't that forcing two `AND` commands, which would fail because `thunar` isn't installed in the apt-get cmd?